app/forms/prior_authority/steps/primary_quote_form.rb

```diff
@@ -0,0 +1,31 @@
+module PriorAuthority
+  module Steps
+    class PrimaryQuoteForm < ::Steps::BaseFormObject
+      attribute :service_name, :value_object, source: QuoteServices
+      attribute :contact_full_name, :string
+      attribute :organisation, :string
+      attribute :postcode, :string
+
+      validates :service_name, presence: true
+      validates :contact_full_name, presence: true, format: { with: /\A[a-z,.'\-]+( +[a-z,.'\-]+)+\z/i }
+      validates :organisation, presence: true
+      validates :postcode, presence: true, uk_postcode: true
+
+      def service_name_suggestion=(value)
+        self.service_name = value
+      end
+
+      private
+
+      def persist!
+        record.update!(attributes.merge(attributes_to_reset))
+      end
+
+      def attributes_to_reset
```

Comment:
I think this  method name is wrong - we're not resetting anything, that's the terminology we use when a change to this form causes us to want to delete now-invalid attributes related to another form. Maybe 'default_attributes' would be a better name here.